src/main/java/org/prebid/server/bidder/BidderCatalog.java

```diff
@@ -19,7 +22,7 @@
 
     private final Map<String, BidderDeps> bidderDepsMap;
 
-    private final Map<String, String> deprecatedNameToError = new HashMap<>();
+    private final Map<String, BidderErrorInfo> deprecatedNameToError = new HashMap<>();
```

Comment:
In order to avoid mixing wire-protocol related specifics with "business" logic I suggest keeping this map as it was previously. It's `ExchangeService#extractDeprecatedBiddersErrors` who will  be convert them to `ExtBidderError`.